packages/web-scripts-utils/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@spotify/web-scripts-utils",
+  "version": "2.0.0",
+  "private": true,
```

Comment:
private dependency means it should not be published when everything else is published, it just is packaged in at release (hopefully).